astropy/io/fits/convenience.py

```diff
@@ -442,6 +442,12 @@ def table_to_hdu(table):
     ----------
     table : astropy.table.Table
         The table to convert.
+    astropy_native : bool
```

Comment:
Same comment about breaking into a longer more narrative part and then a short description of the keyword in the Parameters section.